README.md

```diff
@@ -51,6 +51,67 @@ pandas-specific things that are right now hard-coded in the theme. We also need
 to work on better configurability and extensibility. Feedback and contributions
 are very welcome!
 
+## Theme development
+
+The theme is bundled via Webpack. In `./src/*` the theme related stylesheets and javascript is located. 2 entrypoints are available:
+
+- Stylesheet: `./src/scss/theme.scss'
+- Javascript: `./src/js/theme.js`
+
+Both entrypoints will be bundled into `./pandas_sphinx_theme/static/{css,js}`.
+
+Theme development was inspired by [ReadTheDocs](https://github.com/readthedocs/sphinx_rtd_theme).
+
+### Steps to develop the theme:
+
+1. Install yarn
+2. Install theme dependencies
+3. Run development server
+4. Build production assets
+
+#### Install yarn
+
+[Yarn](https://yarnpkg.com) is a package manager we use for managing Javascript and CSS dependencies.
+
+Install via conda:
+
+```bash
+conda install
+```
+
+Install alternative: https://classic.yarnpkg.com/en/docs/install#mac-stable.
+
+#### Install theme dependencies
+
+Install theme related dependencies:
+
+```bash
+yarn install
+```
+
+#### Run development server
+
+```bash
+yarn build:dev
+```
+
+A development server is availlable at http://localhost:1919. When working on the theme, like editing stylesheets, javascript, .rst or .py files every saves reloads the development server. This reload includes bundling stylesheets, javascript and run sphinx again.
```

Comment:
```suggestion
A development server is available at http://localhost:1919. When working
on the theme, like editing stylesheets, javascript, .rst or .py files
every save reloads the development server. This reload includes bundling
stylesheets, javascript, and runing sphinx again.
```

Reply:
What if running Sphinx is an expensive operation? E.g., the matplotlib docs take like 5 minutes to build from scratch if one were developing a theme...

Reply:
I think in theme development we are able to limit the extensive operations? In real documentation environments, the theme will be used, not developed. Am I correct on this one?